app/views/qsets/_modal_question.erb

```diff
@@ -87,5 +87,9 @@
       var $question_link = $(event.relatedTarget); // element that triggered the modal. $question_link is populated with the data from the click
       initQuestionDisplayModal($(this), $question_link);
     })
+
+    $('#question_display_Modal').on('shown.bs.modal', function (event) {
+      $('textarea').focus();
```

Comment:
Note this will work because there are no other `textarea` elements anywhere. If there were, this would not work as expected, because this selector would return a list of the `textarea` elements on the page. So this works, but it would be best to be more specific -- given there's only one quiz-me modal on the page we could attach a unique `id` attribute to that textarea element and select it here instead of this more general selector. It would future-proof us.